build/build-release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+set -x
+
+readonly REPO_ROOT=$(git rev-parse --show-toplevel)
+
+sudo rkt run --uuid-file-save=rkt.uuid \
```

Comment:
I left the build process exactly the same, as what was done in `build/build-image.sh`, just without the docker build and docker push commands I had to edit / comment whenever I used it.